R/summariseOmopTable.R

```diff
@@ -22,133 +22,106 @@ summariseOmopTable <- function(omopTable,
                                recordsPerPerson = c("mean", "sd", "median", "q25", "q75", "min", "max"),
                                inObservation = TRUE,
                                standardConcept = TRUE,
-                               sourceConcept = FALSE,
+                               sourceVocabulary = FALSE,
                                domainId = TRUE,
                                typeConcept = TRUE) {
-  # initial checks
+
+  # Initial checks ----
   assertClass(omopTable, "omop_table")
+
   omopTable |>
     omopgenerics::tableName() |>
     assertChoice(choices = tables$table_name)
+
   estimates <- PatientProfiles::availableEstimates(
-    variableType = "numeric", fullQuantiles = TRUE
-  ) |>
+    variableType = "numeric", fullQuantiles = TRUE) |>
     dplyr::pull("estimate_name")
   assertChoice(recordsPerPerson, choices = estimates, null = TRUE)
+
   recordsPerPerson <- unique(recordsPerPerson)
+
   assertLogical(inObservation, length = 1)
   assertLogical(standardConcept, length = 1)
-  assertLogical(sourceConcept, length = 1)
+  assertLogical(sourceVocabulary, length = 1)
   assertLogical(domainId, length = 1)
   assertLogical(typeConcept, length = 1)
 
   if ("observation_period" == omopgenerics::tableName(omopTable)) {
-    standardConcept <- FALSE
-    sourceConcept <- FALSE
-    domainId <- FALSE
+    if(standardConcept){
```

Comment:
for the ones that the default is true add:
`if (standardConcept && missing(standardConcept))`
so the warning is only displayed if the user explicitly set it to TRUE